tony-core/src/test/java/com/linkedin/tony/TestEventHandler.java

```diff
@@ -53,12 +53,13 @@ public void setup() {
   }
 
   @Test
```

Comment:
Generally, you put the test for a class in the same package as the class itself. So since EventHandler is in package `com.linkedin.tony.events`, TestEventHandler should be in the same package under the test folder. By doing this, you can make the `@VisibleForTesting` methods package-private instead of public.